docs/streaming-programming-guide.md

```diff
@@ -1415,6 +1415,95 @@ Note that the connections in the pool should be lazily created on demand and tim
 
 ***
 
+## Accumulator and Broadcast
+
+Accumulator and Broadcast cannot be recovered from checkpoint in Streaming. If you enable checkpoint and use Accumulator or Broadcast as well, you have to create lazily instantiated singleton instances for Accumulator and Broadcast so that they can be restarted on driver failures. This is shown in the following example.
```

Comment:
I'd say: "in **Spark** Streaming. If you enable checkpoint**ing** and use an Accumulator or Broadcast as well, you**'ll** have to create ..."